mlrun/package/packagers_manager.py

```diff
@@ -715,17 +718,31 @@ def _unpack_data_item(self, data_item: DataItem, type_hint: type):
     def _look_for_extra_data(
         key: str,
         artifacts: list[Artifact],
+        artifact_uris: dict,
         results: dict,
     ) -> Union[Artifact, str, int, float, None]:
         """
         Look for an extra data item (artifact or result) by given key. If not found, None is returned.
 
-        :param key:       Key to look for.
-        :param artifacts: Artifacts to look in.
-        :param results:   Results to look in.
+        :param key:             Key to look for.
+        :param artifacts:       Artifacts to look in.
+        :param artifact_uris:   Artifacts URIs to look in.
+        :param results:         Results to look in.
 
-        :return: The artifact or result with the same key or None if not found.
+        :return: The artifact or artifact uri or result with the same key or None if not found.
```

Comment:
Now this comment is incorrect, you are returning the artifact here, not the URI.